### ISSUE TITLE
Add E2E for oe23.03

### ISSUE THREAD

**Issue by Okabe-Rintarou-0** [kmesh-net/kmesh#619]
**What would you like to be added**:
Maybe we need to provided a way to do e2e test/ut in oe23.03(with enhanced kernel)
**Why is this needed**:

**Comment by YaoZengzeng** [comment]
AFAIK, it should customize the CI machine, seems not easy. If you are interested, maybe you could help figure it out :)

**Comment by Okabe-Rintarou-0** [comment]
Maybe create a special build container using openeuler 23.03(Current kmesh-build uses a linux image.). Then build a kernel-enhanced image, push to registry, then run some other tests.  We can use go:build enhanced to separate the tests from others.

**Comment by Okabe-Rintarou-0** [comment]
We will need this for testing in ads mode.

**Comment by YaoZengzeng** [comment]
The container and the host share the same kernel. AFAIK, if you want use the kernel enchanced features, you should customize the host kernel, which means you should replace the CI machine's kernel.

Or you just want to run it locally without integrating it into github CI?

**Comment by Okabe-Rintarou-0** [comment]
yes, ur right. It's non-trivial anyway. https://github.com/Yikun/yikun.github.com/issues/85 provides a way to run in custom host, but it will cause security problem i think.
